core/network/src/protocol.rs

```diff
@@ -493,10 +666,16 @@ impl<B: BlockT, S: NetworkSpecialization<B>, H: ExHashT> Protocol<B, S, H> {
 			blocks: blocks,
 		};
 		trace!(target: "sync", "Sending BlockResponse with {} blocks", response.blocks.len());
-		self.send_message(io, peer, GenericMessage::BlockResponse(response))
+		self.send_message(peer, GenericMessage::BlockResponse(response))
 	}
 
-	fn on_block_response(&self, io: &mut SyncIo, peer: NodeIndex, request: message::BlockRequest<B>, response: message::BlockResponse<B>) {
+	fn on_block_response(
+		&mut self,
+		peer: NodeIndex,
+		request: message::BlockRequest<B>,
+		response: message::BlockResponse<B>,
+	) {
+		// TODO: validate response
```

Comment:
is this a new TODO?

Reply:
Actually, this TODO has already been removed on master..